lib/jhipsteruml.js

```diff
@@ -36,12 +37,14 @@ if (options.displayVersion) {
   process.exit(0);
 }
 
-if (!fs.statSync('./.yo-rc.json').isFile()) {
-  console.info(
+fs.exists('./.yo-rc.json', function (exists) {
```

Comment:
It might pop-up anytime as it's asynchronous... perhaps encapsulating what comes next would be better. Anyway, I'd rather not keep it.